sim/src/main/makefrag/midasexamples/Makefrag

```diff
@@ -58,15 +58,19 @@ $(FIRRTL_FILE) $(ANNO_FILE): $(chisel_srcs) $(FIRRTL_JAR) $(SCALA_BUILDTOOL_DEPS
 # Driver Sources & Flags #
 ##########################
 
-driver_dir = $(firesim_base_dir)/src/main/cc
-firesim_lib_dir = $(firesim_base_dir)/firesim-lib/src/main/cc/
-DRIVER_H = $(shell find $(driver_dir) -name "*.h")
+DRIVER_DIR = $(firesim_base_dir)/src/main/cc
```

Comment:
fwiw, these were left lowercase because they are not used as overrides locally, or another make fragment or makefile.